java/src/main/java/org/eclipse/ditto/client/live/events/internal/ImmutableThingEventFactory.java

```diff
@@ -180,6 +180,7 @@ public FeaturePropertyModified featurePropertyModified(final String featureId,
     }
 
     @Override
+    @Deprecated
```

Comment:
Same as with ImmutableGlobalEventFactory. I think we shouldn't deprecate this method. It is not directly tied to V1 and if we introduce a V3 one day, we need it again.